Dafne

dafnevk opened the pull request:
Added pages for Dafne, Elena, and Salient region detectors.

blootsvoets commented on _software/salient-region-detectors.md:
```diff
@@ -0,0 +1,41 @@
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/e.ranguelova
+owner:
+- http://software.esciencecenter.nl/organization/nlesc/
```
remove slash at the end

blootsvoets commented on _software/salient-region-detectors.md:
```diff
@@ -0,0 +1,41 @@
+expertise:
+- Computer Vision
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/e.ranguelova
```
add contact person of

blootsvoets commented:
@elboyran could you review

elboyran commented:
I'm not sure how to do this, I wasn't at the hands-on session. Maybe tomorrow I can talk to whoever is in the center (best- a reviewer ;-).

elboyran commented:
In the project description I would use singular "Software package", not "packages" as it is now. Maybe also it's better to use "detector/detection" rather than "finding". 

I can't seem to open the link of the project, i..e I get 404 File Not Found trying http://software.esciencecenter.nl/software/salient-region-detectors
So, I actually can't review the project page at the moment.

For the people's pages: for my page I would edit the last sentence: "Elena joined..." after "to work on"
and replace the remainder of the sentence with "multidisciplinary projects  with computer vision and machine learning components".
Also from my page there should be a link to Biomarker boosting project!

Dafne's page: Would be nice to have a photo :-). A link not only to the Salient regions, but also to the protein's project?

dafnevk commented:
* WIP means Work in Progress (see http://www.repostatus.org). We haven't released the package yet, that's why I put WIP.
* I changed the words to 'package' , although I would say this is actually specific for python. The software is also implemented in Matlab.
* Thre is no page for the biomarker boosting project, I let it up to someone else to add that.
* I will put my photo up as soon as I get a photo for the website.
* There is no project yet for the Protein Bindings, and I don't think we will put it there in this early stage. 

The other comments I took care of. The branch now passes the validation.
